<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,7,1,2,1">
   <item>
    <widget class="QWidget" name="widget" native="true"/>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QToolButton" name="toolButtonMap1">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="toolButtonMap2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="toolButtonMap3">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="toolButtonMap4">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="toolButtonMap5">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="toolButtonMap6">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map6</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="toolButtonMap7">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="toolButtonMap8">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="toolButtonMap9">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>Map9</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButtonCharacter">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>character</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonBackpack">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>backpack</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonEquipment">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>equipment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonPets">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>pets</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonSkill">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>skill</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>539</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
